### PR TITLE
requested changes

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Comment by cqc-melf** [comment]
@cqc-alec made a new PR to continue the discussion

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -41,11 +42,13 @@ jobs:
       with:
         python-version: '3.9'
     - name: Build and test including remote checks (3.9) mypy
+      shell: bash
       if:  (matrix.os == 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))
       run: |
         ./.github/workflows/build-test mypy
     - name: Build and test including remote checks (3.9) nomypy
       if:  (matrix.os != 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))    
+      shell: bash
```
Comment (cqc-melf) [inline .github/workflows/build_and_test.yml:51]: This is something we need to change on pytket-quantinuum, iqm, qujax, pennylane and braket, too?

File: README.md [hunk 2]
```diff
@@ -68,8 +66,8 @@ script requires `mypy` 0.800 or above.
 #### Linting
 
 We use [pylint](https://pypi.org/project/pylint/) on the CI to check compliance
-with a set of style requirements (listed in `modules/.pylintrc`). You should run
-`pylint` over any changed files from the `modules` directory before submitting a
+with a set of style requirements (listed in `.pylintrc`). You should run
```
Comment (cqc-melf) [inline README.md:69]: same with this? See above

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (cqc-alec) [reply .github/workflows/build_and_test.yml:51]: This fix has already been made to some of them, though perhaps not all.

File: README.md (hunk 2 above)
Comment (cqc-alec) [reply README.md:69]: Possibly, hadn't noticed this before.

**Review by cqc-alec — COMMENTED** [review]
The `pytest.ini` line was needed? https://github.com/CQCL/pytket-aqt/pull/1#discussion_r988078723

**Comment by cqc-alec** [comment]
pytket 1.7 not yet released (stuck because of github actions problems...)

**Comment by cqc-melf** [comment]
> pytket 1.7 not yet released (stuck because of github actions problems...)

Yes I know, I had the hope that it would be solved until tomorrow morning. Maybe a bit to optimistic...

**Comment by cqc-alec** [comment]
Oh dear, this is worrying.

**Comment by cqc-melf** [comment]
will update again to pytket 1.7 after the release is out.

File: docs/changelog.rst [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Changelog
 ~~~~~~~~~
 
-0.24.0 (September 2022)
------------------------
+0.24.0 (October 2022)
+---------------------
 
 * Updated pytket version requirement to 1.7.
```
Comment (cqc-alec) [inline docs/changelog.rst:7]: Should change this too,

Comment (cqc-melf) [reply docs/changelog.rst:7]: Had the plan just to wait merging this PR until the release is out and update the pytket version. Or do you mean something else?

Comment (cqc-alec) [reply docs/changelog.rst:7]: OK, that's fine then.

**Review by cqc-alec — APPROVED** [review]

**Pull request closed without merging**